google/generativeai/answer.py

```diff
@@ -354,6 +358,6 @@ async def generate_answer_async(
         answer_style=answer_style,
     )
 
-    response = await client.generate_answer(request)
+    response = await client.generate_answer(request, **request_options)
```

Comment:
I guess you found this because the test works? LGTM.

Reply:
yes! :)